testsuite/features/secondary/srv_organization_credentials.feature

```diff
@@ -5,14 +5,12 @@ Feature: Organization credentials in the Setup Wizard
 
 @scc_credentials
   Scenario: Enter valid SCC credentials
-    Given I am on the Admin page
```

Comment:
I think we need at least:

Given I am authorized as "admin" with password "admin"

or we keep it as you changed this below as well.

Same for all the below.

Reply:
A-ha! You're right, it was making sure the user has the correct permissions in the previous version.

What about adding a parametrized menu section check?

`Given I am authorized for the "Admin" section`

and then the implementation

```
Given(/^I am authorized for the "([^"]*)" section$/) do |section|
  case section
  when 'Admin'
    step %(When I am authorized as "admin" with password "admin")
  end
end
```

Reply:
I implemented it as described. Could you please take a second look? Thank you